digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -2047,10 +2091,22 @@ class ProjectService(roadAddressService: RoadAddressService, roadLinkService: Ro
       nodesAndJunctionsService.calculateNodePointsForNodes(nodeIds, username)
     }
 
-    if (newState != Saved2TR) {
-      logger.error(s" Project state not at Saved2TR")
-      throw new RuntimeException(s"Project state not at Saved2TR: $newState")
+    projectDAO.fetchProjectStatus(projectID) match {
```

Comment:
Miksi tässä vaiheessa pitää tarkistaa projektin status? Eikös tämän kutsu ole vain tietyssä vaiheessa projektin käsittelyä?

Reply:
Hyvä kysymys - ja vastauksesta ei mitään hajua. Tuohan on siis siellä kaninkolon perukoilla, missä projektin datat viedään kantaan.
Siellä se tarkistus oli nyt jo "alunperin", mää vaan päädyin vähän refaktoroimaan sitä (eli jos se kerran tsekataan, voisi sentään tsekata jotain oikeaa, eikä annettua randomarvoa...).

Miksi... Veikkaan, että sitä on joskus kutsuttu epäsopivassa tilassa syytä tai toisesta, jolloin toi ois sit jonkinlainen vartija sille, ettei vahingossa viedä höpöhöpöjä kantaan - projektin jutut voi kuitenkin viedä kantaan vain kerran.

Lähdetäänkö positamaan? (Kiva, taas näitä tyyliin "kun lauseen muotoiluun on käytetty tunti, joku ehdottaa koko kappaleen pois jättämistä". :D )